app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -81,6 +81,21 @@ func (ctw ConsensusTopicWatcher) getTimestamp(q *queue.Queue) string {
 	return milestoneTimestamp
 }
 
+func (ctw ConsensusTopicWatcher) updateTimestamp(message []byte, timestamp string, q *queue.Queue) {
```

Comment:
since we are not only updating the Timestamp, but actually publishing to the channel as-well (I would say the core of the logic), I think that a better name to the method would be `processMessage`